src/main/java/com/spring/book/management/security/AuthenticationService.java

```
@@ -0,0 +1,29 @@
+package com.spring.book.management.security;
+
+import com.spring.book.management.dto.UserLoginRequestDto;
+import com.spring.book.management.dto.UserLoginResponseDto;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AuthenticationService {
+
+    private final AuthenticationManager authenticationManager;
+    private final JwtUtil jwtUtil;
+
+    public AuthenticationService(AuthenticationManager authenticationManager,
+                                 JwtUtil jwtUtil) {
+        this.authenticationManager = authenticationManager;
+        this.jwtUtil = jwtUtil;
+    }
+
+    public UserLoginResponseDto authenticateUser(UserLoginRequestDto request) {
+        Authentication authentication = authenticationManager.authenticate(
```

Missing error‐mapping - If credentials are bad, `authenticate(...)` throws `BadCredentialsException`. You should catch it and rethrow (or let it bubble) so your global handler can map it to a `401 Unauthorized` rather than a `500`.